client/hwctl/src/main.rs

```diff
@@ -27,7 +27,16 @@ use hwlib::{
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    let cert_status_request = CertificationStatusRequest::new(Paths::default())?;
+    let cert_status_request = match CertificationStatusRequest::new(Paths::default()) {
+        Ok(request_data) => request_data,
+        Err(e) => {
+            eprintln!("failed to collect system data: {}", e);
+            eprintln!(
+                "ensure you're root and running the program on a machine with SMBIOS available"
```

Comment:
I think this advice isn't very extensible, as the problem the user encounters may not relate to the user being the root or the availability of SMBOIS data, in which case this is misleading. I think in this case, it's okay to rely on the user being smart enough to see something like `cannot access file foo: permission denied` and understand that they need to run as the root. In the case of missing SMBIOS data, I don't think there's anything the user can do apart from buy a new computer? :P 

Minor note: I think these two messages use two names for the same thing---are 'system data' and 'SMBIOS' equivalent in the eyes of hwctl? If so, I think it'd be better to use only one name to reduce cognitive overhead on the user

Just for future reference, in these kind of advice / error messages, I think it's good practice to avoid the second person and instead to focus on the program itself as addressing the user directly in this way feels informal. In this case, we could use:
```
ensure that hwctl is running as root
```